tests/test_api_build.py

```diff
@@ -123,6 +124,9 @@ def test_recipe_builds(
     #   ``source_setup_py_data_subdir`` reproduces the problem.
     if recipe.name == "source_setup_py_data_subdir":
         pytest.xfail("Issue related to #3754 on conda-build.")
+    elif recipe.name == "unicode_all_over" and context.solver == "libmamba":
+        pytest.xfail("Unicode package names not supported in libmamba.")
```

Comment:
Yikes

Reply:
libmamba doesn't support unicode package names IIUC, no need to test for it.